File: q431/helloworld.dart
#!/usr/bin/env dart
main(){
	var szero=[].length.toString().length;
	//var szero=-~[];
	//var szero=-~false;
	//var szero;
	//szero=-~szero;
	var sone=szero<<szero;
	var stwo=szero<<(szero+szero);
	var sthree=szero<<(szero+szero+szero);
	var sfour=szero<<(szero+szero+szero+szero);
	var sfive=szero<<(szero+szero+szero+szero+szero);
	var ssix=szero<<(szero+szero+szero+szero+szero+szero);
	var str=new String.fromCharCode(ssix|sthree);
	str+=new String.fromCharCode(ssix|sfive|stwo|szero);
	str+=new String.fromCharCode(ssix|sfive|sthree|stwo);
	str+=new String.fromCharCode(ssix|sfive|sthree|stwo);
	str+=new String.fromCharCode(ssix|sfive|sthree|stwo|sone|szero);
	str+=new String.fromCharCode(sfive);
	str+=new String.fromCharCode(ssix|sfour|stwo|sone|szero);
	str+=new String.fromCharCode(ssix|sfive|sthree|stwo|sone|szero);
	str+=new String.fromCharCode(ssix|sfive|sfour|sone);
	str+=new String.fromCharCode(ssix|sfive|sthree|stwo);
	str+=new String.fromCharCode(ssix|sfive|stwo);
	print(str);
}

File: q431/helloworld.c
//usr/bin/env picoc $0 $@;exit
#include <stdio.h>
int szero,zero;
int main(){
	//int one;
	//one=-~(one^one);
	//one=sizeof(char);
	szero++;
	int sone=szero<<szero;
	int stwo=szero<<(szero+szero);
	int sthree=szero<<(szero+szero+szero);
	int sfour=szero<<(szero+szero+szero+szero);
	int sfive=szero<<(szero+szero+szero+szero+szero);
	int ssix=szero<<(szero+szero+szero+szero+szero+szero);

	putchar(ssix|sthree);
	putchar(ssix|sfive|stwo|szero);
	putchar(ssix|sfive|sthree|stwo);
	putchar(ssix|sfive|sthree|stwo);
	putchar(ssix|sfive|sthree|stwo|sone|szero);
	putchar(sfive);
	putchar(ssix|sfour|stwo|sone|szero);
	putchar(ssix|sfive|sthree|stwo|sone|szero);
	putchar(ssix|sfive|sfour|sone);
	putchar(ssix|sfive|sthree|stwo);
	putchar(ssix|sfive|stwo);
	putchar(sthree|sone);
	return zero;
}